packages/appstore/src/components/account/account.tsx

```diff
@@ -1,56 +1,276 @@
-import { Localize } from '@deriv/translations';
+import { Text, Button, Icon, Money } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
 import WalletIcon from 'Assets/svgs/wallet';
-import { getCurrencyDisplayCode } from '@deriv/shared';
+import { getCurrencyDisplayCode, isMobile, getCurrencyName, getCFDAccountDisplay, isBot } from '@deriv/shared';
+import classNames from 'classnames';
+import { RouteComponentProps } from 'react-router';
 
-type TOptionsAccountprops = {
-    account_icon: string;
-    account_title?: string;
-    account_number?: string;
-    account_balance?: string;
+interface Icountry_standpoint {
+    is_united_kingdom: string;
+    is_isle_of_man: string;
+}
+
+interface Igeolocation {
+    region: any;
+    sequence: number;
+}
+interface Iserver_info {
+    geolocation: Igeolocation;
+}
+interface Iserver {
+    server_info: Iserver_info;
+}
+
+type TOptionsAccountprops = RouteComponentProps & {
+    currency_icon: string | undefined;
+    loginid_text: string;
+    balance?: string;
     account_button?: string;
     currency?: string;
+    display_type: string;
+    has_balance?: boolean;
+    has_reset_balance?: boolean;
+    is_disabled?: boolean;
+    is_virtual?: boolean;
+    title?: string;
+    country_standpoint: Icountry_standpoint;
+    is_eu?: string;
+    market_type?: string;
+    server?: Iserver;
+    sub_account_type?: string;
+    has_error?: string;
+    platform?: string;
+    is_dark_mode_on?: string;
+    shortcode?: string;
+    should_show_server_name?: string;
+    onClickResetVirtualBalance: () => void;
+    selected_loginid?: string;
+    redirectAccount: () => void;
+    activeAccount?: string;
+    onClickDeposit?: () => void;
+};
+
+type TCurrentDisplay = {
+    country_standpoint: Icountry_standpoint;
+    currency?: string;
+    loginid: string;
+    is_virtual?: boolean;
+};
+
+type TAccountDisplay = {
+    is_eu?: string;
+    market_type?: string;
+    server?: Iserver;
+    sub_account_type?: string;
+    has_error?: string;
+    platform?: string;
+    is_dark_mode_on?: string;
+    shortcode?: string;
+    should_show_server_name?: string;
 };
 
 const OptionsAccount = ({
-    account_icon,
-    account_title,
-    account_number,
-    account_balance,
-    account_button,
+    currency_icon,
+    loginid_text,
+    balance,
     currency,
+    display_type,
+    has_reset_balance,
+    is_disabled,
+    is_virtual,
+    country_standpoint,
+    is_eu,
+    market_type,
+    server,
+    sub_account_type,
+    has_error,
+    platform,
+    is_dark_mode_on,
+    shortcode,
+    should_show_server_name,
+    onClickResetVirtualBalance,
+    redirectAccount,
+    activeAccount,
+    onClickDeposit,
 }: TOptionsAccountprops) => {
+    const currency_badge = currency ? currency_icon : 'IcCurrencyUnknown';
     return (
-        <div className='account__container'>
+        <div
+            className={`account__container ${
```

Comment:
Change the class name to account-container and also You can use classNames here which is the better approach.
className={classNames('account-container', {
                'account-container--active': activeAccount === loginid_text,
                'account-container--disabled': ...
})}

Reply:
Thanks , Changed to this approach.